config/initializers/fog.rb

```diff
@@ -0,0 +1,11 @@
+CarrierWave.configure do |config|
+  config.fog_credentials = {
+    :provider               => 'AWS',                        # required
+    :aws_access_key_id      => ENV['AWS_ACCESS_KEY_ID'],                        # required
+    :aws_secret_access_key  => ENV['AWS_SECRET_ACCESS_KEY'],                        # required
```

Comment:
Line is too long. [94/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.